packages/hiro-graph-orm/index.d.ts

```diff
@@ -13,6 +13,7 @@ export interface IDefinition {
   required?: IDefinitionData;
   optional?: IDefinitionData;
   relations?: IDefinitionData;
+  [name: string]: string | IDefinitionData | undefined;
```

Comment:
I wonder if `[name: string]?: string | IDefinitionData;` would work

Reply:
It wouldn't. :( [Check here](https://www.typescriptlang.org/play/index.html#code/MYGwhgzhAEDiCmAXR8BO0DeBYAUNaA5kgCJiJgAUAlAFzQSKoCWAdgZrvvqkgK6otoAckgATAGZCA3J2gBfXApy5WKVOLDB40ALIBPAMLgoHPIRJlKBOgmRpa9RqwIyzRRKXLU6DZm1dKKixqGlrQACrwDKb4APYs8D5O-rKIAO6xdCy8ALYARmiu+ADaANbwekl+BAC6APxVztAAPtDZ+YWKuLjA8dE6OnT6RpAwALwx0KGIsah63rqGxjDYZtx8AtCIABZMEEXy3W4WXtZwSGoODE0Y0LLriPyCBAB07p6UVLJKgTi9LNEcsBoBMEmlFiMoNQen1YiB4C8QLECBRhss3idPl8-rD4YjkRQgRiPJZqFQgA)